### PR TITLE
OW1Ka: fix crash if link contains no survey responses

### PR DESCRIPTION
**Pull request by kernc**
Fixes https://github.com/biolab/orange3-prototypes/issues/92

### CODE REVIEW

File: orangecontrib/prototypes/widgets/ow1ka.py [hunk 1]
```diff
@@ -255,7 +268,10 @@ def apply_domain_edit(self):
 
     def table_from_html(self, html):
         soup = BeautifulSoup(html, 'html.parser')
-        html_table = soup.find_all('table')[-1]
+        try:
+            html_table = soup.find_all('table')[-1]
+        except IndexError:
+            raise DataEmptyError
```
Comment (astaric) [inline orangecontrib/prototypes/widgets/ow1ka.py]: I'd add (). We usually raise Exception instances, not classes.

Comment (kernc) [reply orangecontrib/prototypes/widgets/ow1ka.py]: I prefer it this way when the exception has no arguments. It's sugar. https://docs.python.org/3/tutorial/errors.html#raising-exceptions Regex `raise \w+$` shows our use of it is quite broad.

Comment (astaric) [reply orangecontrib/prototypes/widgets/ow1ka.py]: I love the "one obvious way to do it" approach python has :) Leave it as it is then

**Comment by ajdapretnar** [comment]
The submitted survey still doesn't work with this fix. Seems like the issue was not an empty survey, but there must be a None elsewhere... Anyhow, I have results now and I can't see them in Orange.

**Comment by kernc** [comment]
The changeset _fixes_ the crash when the result data is empty, such as in https://www.1ka.si/podatki/141045/63A99D0B/.

It _doesn't fix_ the NoneType crash due to incompatible (unprecedented!) result set format, such as in https://www.1ka.si/podatki/141025/5730371E/ Dafuq is that?!

**Comment by kernc** [comment]
Should not longer crash on Analysis link.

**Pull request merged**